db/migrate/20240422123545_add_service_to_feedback.rb

```diff
@@ -0,0 +1,5 @@
+class AddServiceToFeedback < ActiveRecord::Migration[7.1]
+  def change
+    add_column :feedbacks, :service, :string, :default => :check, null: false
```

Comment:
As per previous comment, we should then do the following:

`change_column_default :feedback, :service, nil`